sdk/appconfiguration/app-configuration/test/internal/node/throttlingRetryPolicy.spec.ts

```diff
@@ -58,9 +58,9 @@ describe("Should not retry forever", () => {
       await Promise.all(promises);
```

Comment:
Feel free to ignore but I think you could do something like 

```ts
await expect(Promise.all(promises)).rejects.toThrow(/The operation was aborted/);
```
Kinda like https://github.com/Azure/azure-sdk-for-js/blob/4fc29fe715d4b78bbc276c364d304a28c067267a/sdk/core/core-rest-pipeline/test/throttlingRetryPolicy.spec.ts#L286

Instead of the try/catch + errorWasThrown thing